src/AddTodoForm.js

```diff
@@ -1,10 +1,19 @@
 import React from "react";
 
-function AddTodoForm () {
+function AddTodoForm(props) {
+    const [inputText, setInputText] = React.useState("");
```

Comment:
For now you won't need this useState hook here, since you're using the onAddTodo callback to pass the todoTitle variable.